docs/upgrade_from_v0.11.1_to_v0.25.1.md

```diff
@@ -0,0 +1,187 @@
+| `spec.auth`                                              | `spec.kubernetes.awsAuth`                                                                                                                            ||
+| `spec.logRetentionDays`                                  | `spec.kubernetes.logRetentionDays`                                                                                                                   ||
+| `executor.state.backend.bucket`                          | `spec.toolsConfiguration.terraform.state.s3.bucketName`                                                                                              ||
+| `executor.state.backend.key`                             | `spec.toolsConfiguration.terraform.state.s3.keyPrefix`                                                                                               | On S3 move/rename the ../../../bin/terraform/1.4.6/terraform state file to `cluster.json` under the folder specified by `keyPrefix` |
+| `executor.state.backend.region`                          | `spec.toolsConfiguration.terraform.state.s3.region`                                                                                                  ||
+
+> NOTE: in case you have only the `cluster.yml`, set `spec.kubernetes.subnetsIds` on `furyctl.yml`
+
+The following `distribution` fields are the minimum required to be present on `furyctl.yml`. 
+
+Adjust value according your environment
+
+```yaml
+spec:
+  distribution:
+    common:
+      nodeSelector:
+        node.kubernetes.io/role: infra
+      tolerations:
+        - effect: NoSchedule
+          key: node.kubernetes.io/role
+          value: infra
+    modules:
+      ingress:
+        baseDomain: internal.example.dev
+        nginx:
+          type: dual
+        dns:
+          public:
+            name: "example.dev"
+            create: false
+          private:
+            name: "internal.example.dev"
+            create: false
+      logging:
+        type: opensearch
+        opensearch:
+          type: single
+      policy:
+        type: gatekeeper
+      dr:
+        type: none
+        velero:
+          eks:
+            bucketName: example-velero
+            region: eu-west-1
+```
+
+6. Using the furyctl v0.25.1 
+```shell
+#Ensure your aws credential has been set
+furyctl create cluster --dry-run --phase infrastructure -o $(pwd)
+
+# where METADATA_NAME is the value of `metadata.name` in `furyctl.yml`
+cd .furyctl/METADATA_NAME/infrastructure/terraform
+
+export VPC_AND_VPN_MODULE_NAME=vpc-and-vpn
+export VPC_MODULE_NAME='vpc[0]'
+export VPN_MODULE_NAME='vpn[0]'
+#export VPN_INSTANCES=2
+#export LENGTH_PRIVATE_SUBNETS=3
+#export LENGTH_PUBLIC_SUBNETS=3
+
+echo "../../../bin/terraform/1.4.6/../../../bin/terraform/1.4.6/terraform state mv 'module.${VPC_AND_VPN_MODULE_NAME}.module.vpc.aws_vpc.this[0]' 'module.${VPC_MODULE_NAME}.module.vpc.aws_vpc.this[0]'" | sh
```

Comment:
what is the reason for this `../../../bin/terraform/1.4.6/../../../bin/terraform/1.4.6/` path traversing? Is it a wrong find+replace?

Reply:
The reason is this line https://github.com/sighupio/furyctl/pull/421/files/9be585868d79cf5413564d3c6dbf260e36a55ca7#diff-42e7d50a282a0ca757f73e38e60e4cce5f2eda2f6fcac43ee9803afee42b6e4eR114